dockstore-webservice/src/main/java/io/dockstore/webservice/core/AppTool.java

```diff
@@ -26,7 +26,12 @@
 @Entity
 @Table(name = "apptool")
 @NamedQueries({
-    @NamedQuery(name = "io.dockstore.webservice.core.AppTool.getEntriesByUserId", query = "SELECT a FROM AppTool a WHERE a.id in (SELECT ue.id FROM User u INNER JOIN u.entries ue where u.id = :userId)")
+    @NamedQuery(name = "io.dockstore.webservice.core.AppTool.getEntriesByUserId", query = "SELECT a FROM AppTool a WHERE a.id in (SELECT ue.id FROM User u INNER JOIN u.entries ue where u.id = :userId)"),
```

Comment:
curious, is this used anywhere or is this just added out of consistency?

Reply:
It's called from [here](https://github.com/dockstore/dockstore/blob/1bf8045a58fdec2ec66d3e98bdfb2c6e25cef55c/dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java#L212). I stumbled across it because when I added this line `entriesLite.addAll(appToolDAO.findEntryVersions(userId));` in UserResource I got an error that the query didn't exist.